Avoid using scm_from_complex_double(csqrt()) when building with uclibc.

Signed-off-by: Pedro Aguilar <paguilar@paguilar.org>

diff -Nau guile-2.0.11.orig/configure.ac guile-2.0.11/configure.ac
--- guile-2.0.11.orig/configure.ac	2014-03-12 14:36:02.000000000 +0100
+++ guile-2.0.11/configure.ac	2014-11-03 23:59:51.897267207 +0100
@@ -862,7 +862,7 @@
 }]])],
     [guile_cv_use_csqrt=yes],
     [guile_cv_use_csqrt="no, glibc 2.3 bug"],
-    [guile_cv_use_csqrt="yes, hopefully (cross-compiling)"])])
+    [guile_cv_use_csqrt="no (cross-compiling)"])])
   case $guile_cv_use_csqrt in
     yes*)
       AC_DEFINE([HAVE_USABLE_CSQRT], 1, [Define to 1 if csqrt is bug-free])
